packages/react-core/src/components/Page/examples/Page.md

```diff
@@ -23,49 +23,88 @@ import './page.css';
 
 ```ts file="./PageHorizontalNav.tsx"
 ```
 
-### Tertiary nav
+### Tertiary navigation
+
+To add a tertiary navigation component to a page, use the `tertiaryNav` property. The following example passes a string of ‘Navigation’ to `tertiaryNav`, but you can pass fully built-out components, such as navigation.
+
+In this example, `<Masthead>` sets the `display` property equal to `"{{ default: 'stack' }}"` to stack the masthead items and also passes in `"inset={{ default: 'insetXs' }}"` to adjust the inset padding.
 
 ```ts file="./PageTertiaryNav.tsx"
 ```
 
-### With or without fill
+### Uncontrolled nav
+
+When the `isManagedSidebar` property is true, it manages the sidebar open/close state, removing the need to pass both `isNavOpen` into the `<PageSidebar>` and `onNavToggle` into the `<PageHeader>`.
+
+```ts file="./PageUncontrolledNav.tsx"
+```
+
+### Filled page sections
+
+To make a `<PageSection>` fill the available vertical space of the page, set the `isFilled` property equal to true.
+
+By default, the last page section sets `isFilled` equal to true.
 
 ```ts file="./PageWithOrWithoutFill.tsx"
 ```
 
 ### Main section padding
 
-```ts file="./PageMainSectionPadding.tsx"
-```
+To adjust the padding of a `<PageSection>`, you can pass in different values to the `padding` property. These value should be aligned to various breakpoints, including 'default', 'sm', 'md', 'lg', 'xl', '2xl'. 
 
-### Uncontrolled nav
+To remove padding, pass 'noPadding' into `padding`. For example, the second section in this example passes in `padding={{ default: 'noPadding' }}`. Since no specific breakpoints are mentioned, every breakpoint will have 'noPadding'.
 
-```ts file="./PageUncontrolledNav.tsx"
-```
+To add padding on specific breakpoints, specify the value for those breakpoints. For example, the third section in this example passes in `padding={{ default: 'noPadding', md: 'padding', lg: 'padding' }}`. On 'md' and 'lg' breakpoints, the default value will be overwritten, and padding will be added.
 
-### Group section
+To remove padding at specific breakpoints, pass in 'noPadding' at those breakpoints. For example, the fourth section in this example passes in `padding={{ md: 'noPadding' }}`, which means that 'md' is the only breakpoint that will remove padding.
 
-```ts file="./PageGroupSection.tsx"
+```ts file="./PageMainSectionPadding.tsx"
 ```
 
-### Vertical nav using PageHeader component
+### Group section
 
-This example is provided because PageHeader and PageHeaderTools are still in use; however, going forward Masthead and Toolbar should be used to make headers rather than PageHeader and PageHeaderTools.
+To group page content sections, add one or more `<PageGroup>` components to a `<Page>`. 
 
-```ts file="./PageVerticalNavUsingPageHeaderComponent.tsx"
+The following example adds a group containing both the `<PageNavigation>` and a section of content.
```

Comment:
FWIW this group has `<PageNavigation>`, `<PageBreadcrumb>` and `<PageSection>`. It's also worth noting that you can group things in a `<PageGroup>` via `isBreadcrumbGrouped`, `isTertiaryNavGrouped`, and `additionalGroupedContent `. Not sure if that's relevant for this example, but wanted to mention it.